projects/@shared/components/properties-item.vue

```diff
@@ -171,10 +171,7 @@ export default {
   },
   computed: {
     errorMessage() {
-      if (this.errors && this.errors.length > 0) {
-        return this.errors[0].message;
-      }
-      return null;
+      return this.warnings?.[0]?.message ?? null;
```

Comment:
I think the `?? null` is obsolete when using optional chaining, because as soon as any part doesn't exist it returns `undefined` which works just as well as `null` in our case.